### PR TITLE
Basic Authpages & router setup

### PR DESCRIPTION
**Pull request by rushikesh3299**
Login & signup page UI
Basic react-router-setup

Animation

### CODE REVIEW

File: src/pages/authpages/Login.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./login.css";
+
+function Login() {
+  return (
+    <div class="login-container">
+      <img
+        class="login-img"
```
Comment (tusharkandpal) [inline src/pages/authpages/Login.js]: please convert all `class` attributes to `className`

**Review by tusharkandpal — APPROVED** [review]
so far so good.! 💯

File: src/pages/authpages/Login.js (hunk 1 above)
Comment (rushikesh3299) [reply src/pages/authpages/Login.js]: Done. Thanks for reminding

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
 import "./App.css";
+import { Routes, Route, Link } from "react-router-dom";
 import Navbar from "./components/navbar/Navbar";
+import Login from "./pages/authpages/Login";
+import Signup from "./pages/authpages/Signup";
```
Comment (t007rushi) [inline src/App.js]: make an ```index.js``` in pages export from there

File: src/pages/authpages/Login.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./login.css";
+
+function Login() {
+  return (
+    <div className="login-container">
+      <img
+        className="login-img"
+        src="https://ayumall-screens.netlify.app/Assets/AyuMall.png"
```
Comment (t007rushi) [inline src/pages/authpages/Login.js]: Login page is fixed static so put this in public folder

File: src/pages/authpages/Signup.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./login.css";
+
+function Signup() {
+  return (
+    <div className="login-container">
+      <img
+        className="login-img"
+        src="https://ayumall-screens.netlify.app/Assets/AyuMall.png"
```
Comment (t007rushi) [inline src/pages/authpages/Signup.js]: same as  above

**Review by t007rushi — COMMENTED** [review]
suggested some changes

File: src/App.js [hunk 5]
```diff
@@ -1,10 +1,17 @@
...
 
 function App() {
   return (
     <div className="App">
       <Navbar />
+      <Routes>
+        <Route path="login" element={<Login />} />
+        <Route path="signup" element={<Signup />} />
+      </Routes>
```
Comment (dikshadhote) [inline src/App.js:14]: Can use separate file to define routes to make code modular

File: src/pages/authpages/Login.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+      <h2>WelCome!</h2>
+      <h3>Enter Your info below to Login</h3>
+      <div className="login-inputs">
+        <input type="text" className="userid" placeholder="User ID" />
+        <input type="password" name="passwd" id="" placeholder="Password" />
+        <button className="login-btn">Login</button>
+        <div className="login-footer">
+          <div>
+            <a href="#">Forgot Password?</a>
+          </div>
```
Comment (dikshadhote) [inline src/pages/authpages/Login.js:17]: Forgot Password will be separate page.so, you can use <Link> instead of <a>

**Review by dikshadhote — CHANGES_REQUESTED** [review]

**Pull request merged**